erpnext/accounts/doctype/purchase_invoice/purchase_invoice.json

```diff
@@ -208,6 +208,31 @@
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
+   "default": "0", 
+   "fieldname": "is_cash", 
```

Comment:
Should this be `include_payment` (as payment may not necessarily be "cash", but could be a check?)